config/locales/countries/en.yml

```diff
@@ -0,0 +1,222 @@
+en:
+  countries:
+    ad: Andorra
```

Comment:
QQ: what was the purpose of lowercasing these keys?

Reply:
Oh it was failing the linting where we require keys to be lowercase.